node/network/availability-recovery/src/lib.rs

```diff
@@ -880,6 +886,29 @@ async fn launch_recovery_task<Context>(
 		bypass_availability_store,
 	};
 
+	if let Some(small_pov_limit) = small_pov_limit {
+		// Get our own chunk size to get an estimate of the PoV size.
+		let chunk_size: Result<Option<usize>, error::Error> =
+			query_chunk_size(ctx, candidate_hash).await;
+		if let Ok(Some(chunk_size)) = chunk_size {
+			let pov_size_estimate = chunk_size.saturating_mul(session_info.validators.len() / 3);
```

Comment:
Better do the division on the outer level - should work better with smaller networks.

Reply:
done